frontend/src/components/StockDetails/StockQuote/StockQuote.js

```diff
@@ -34,36 +37,31 @@ class StockQuote extends React.Component {
         return format(date, "MMM dd, yyyy");
     }
     render() {
+        const { c, ...props } = this.state.stockQuote;
+        const t = new Date().getTime() / 1000;
         return (
             <div id="stock-quotes">
-                {
-                    stockData.map((quote, index) => {
-                        const { t, c, ...props } = quote;
-                        return (
-                            <Card key={index} className="quote">
-                                <CardBody>
-                                    <CardTitle className="header">
-                                        <div className="company">Sabre</div>
-                                        <div>{this.convertUnixTimeToDate(t)}</div>
-                                    </CardTitle>
-                                    <p className="price">${c}</p>
-                                    <Row>
-                                        {
-                                            Object.entries(props).map(([key, value]) => (
-                                                <Col key={key}>
-                                                    <div className="font-weight-bold">
-                                                        <u>{key}</u>
-                                                    </div>
-                                                    <div>${value}</div>
-                                                </Col>
-                                            ))
-                                        }
-                                    </Row>
-                                </CardBody>
-                            </Card>
-                        );
-                    })
-                }
+                <Card className="quote">
+                    <CardBody>
+                        <CardTitle className="header">
+                            <div className="company">{this.props.companyName}</div>
+                            <div>{this.convertUnixTimeToDate(t)}</div>
```

Comment:
Remove `<div>{this.convertUnixTimeToDate(t)}</div>` and `convertUnixTimeToDate()`. Make the HTML and CSS for aligning the company name align to the left more concise.

Reply:
What do you mean by making the HTMl/CSS more concise?

Reply:
I meant: can you find a way to shorten the amount of HTML/CSS needed to align the company name to the left?